rootfs/scheduler/__init__.py

```diff
@@ -721,19 +721,18 @@ def _healthcheck(self, controller, path='/', port=8080, delay=30, timeout=1):
     # SECRETS #
 
     def _create_secret(self, namespace):
-        secretId, secretKey = '', ''
-        with open("/var/run/secrets/deis/minio/user/access-key-id") as the_file:
+        with open("/var/run/secrets/deis/minio/user/access-key-id", "rb") as the_file:
             secretId = the_file.read()
-        with open("/var/run/secrets/deis/minio/user/access-secret-key") as the_file:
+        with open("/var/run/secrets/deis/minio/user/access-secret-key", "rb") as the_file:
             secretKey = the_file.read()
-        Key, Id = base64.b64encode(secretKey), base64.b64encode(secretId)
-        l = {
+
+        template = json.loads(string.Template(SECRET_TEMPLATE).substitute({
             "version": self.apiversion,
             "id": namespace,
-            "secretId": Id,
-            "secretKey": Key,
-            }
-        template = json.loads(string.Template(SECRET_TEMPLATE).substitute(l))
+            "secretId": base64.b64encode(secretId).decode(),
```

Comment:
Because that's what the documentation suggests. Originally did `str` but was like "docs must be gooood". I'll change it back to `str` since it is easier to grok it